src/components/publicApi/PublicLoginScreen.tsx

```diff
@@ -4,13 +4,14 @@ import * as React from 'react'
 import { initializeLogin } from '../../actions/LoginInitActions'
 import { updateFontStyles } from '../../constants/Fonts'
 import { Branding, ParentButton } from '../../types/Branding'
-import { OnLogin, OnNotificationPermit } from '../../types/ReduxTypes'
+import { Action, OnLogin, OnNotificationPermit } from '../../types/ReduxTypes'
 import { Router } from '../navigation/Router'
 import { ReduxStore } from '../services/ReduxStore'
 import { changeFont, useTheme } from '../services/ThemeContext'
 
 interface Props {
   context: EdgeContext
+  initialAction?: Action
```

Comment:
Currently, there is no way to get back to the getting started scene after tapping "Skip" to navigate away. I think this is okay though because the welcome scene doesn't give you any extra functionality that the default login scene does. Though, if we wanted to be able to navigate back to the getting started scene, we could implement something like you suggested. However, the word "Skip" implies a replacement of the scene in my mind, rather than a navigate forward (that'd be more like "Continue", or "Next").

I will make the changes you suggest regarding the initialScene prop. That makes more sense. My first attempt was a bit hacky.

I'm not sure I understand what you mean in the 3rd paragraph. Are you suggesting we deprecate the different exported scenes in favor of one exported scene which would require to the use of this new prop to regain the functionality?

Reply:
Yes, that's exactly what I meant. Then we just export one component, which uses a prop to decide where to route the user, instead of exporting half a dozen different components.